### DISCUSSION TITLE
signInAnonymously suddenly giving a new user id each load

### DISCUSSION THREAD

**Discussion by selven** [invertase/react-native-firebase#6585] [Q&A]
Hey, I've been using signInAnonymously in my app for about 6 months and the user id has never changed. But this last week on android I've started to get a new user id every time I force close the app and open it.
Is anyone else experiencing this? I haven't released a new version in over a month so not a bug in the code or firebase sdk.

It seems to be android only and only started happening this week. It means users data is being wiped every time they open the app.

**Comment by mikehardy** [comment]
Based on "seems to be android only" and "I haven't released a new version in over a month so not a bug in the code or firebase sdk", you might head over to firebase-android-sdk and ask them? They will *most likely* be suspicious of react-native-firebase including the react-native layer on top of their SDK, and rightly so, but you can do a quickstart repro from this https://github.com/firebase/quickstart-android/tree/master/auth to prove the point if needed
